### PR TITLE
Issue 4056

### PR DESCRIPTION
**Pull request by atavism**
This resolves https://github.com/getlantern/lantern/issues/4056 and https://github.com/getlantern/lantern/issues/4055

### CODE REVIEW

**Comment by atavism** [comment]
@oxtoacart I forgot to open a PR for this one. I also fixed this on the ``lantern-mobile-pro`` branch (where the code has changed significantly).

**Comment by oxtoacart** [comment]
@atavism Sorry, I'm confused.  This is a PR.  What did you forget to open a PR for?

**Comment by atavism** [comment]
This PR to resolve #4056 and #4055.... I opened the two others but not this one?

**Comment by atavism** [comment]
The test that is failing:
```
--- FAIL: TestProxying (5.06s)
	Error Trace:	lantern_test.go:28
	Error:		Received unexpected error "HTTP Proxy didn't start within given timeout"
	Messages:	Should have been able to start lantern
```

File: src/github.com/getlantern/lantern-mobile/app/build.gradle [hunk 1]
```diff
@@ -99,6 +99,7 @@ dependencies {
     compile "com.android.support:appcompat-v7:${supportLibVersion}"
     compile "com.android.support:design:${supportLibVersion}"
 
+    compile 'commons-io:commons-io:2.4'
```
Comment (oxtoacart) [inline src/github.com/getlantern/lantern-mobile/app/build.gradle]: Why is this needed?

Comment (atavism) [inline src/github.com/getlantern/lantern-mobile/app/build.gradle]: The mandrill library depends on it:

```
              LanternUI  E  Failed to parse response from request 'https://mandrillapp.com/api/1.0/messages/send-template.json'
                         E  java.io.IOException: Failed to parse response from request 'https://mandrillapp.com/api/1.0/messages/send-template.json'
                         E      at com.microtripit.mandrillapp.lutung.model.MandrillRequestDispatcher.execute(MandrillRequestDispatcher.java:86)
                         E      at com.microtripit.mandrillapp.lutung.controller.MandrillUtil.query(MandrillUtil.java:46)
                         E      at com.microtripit.mandrillapp.lutung.controller.MandrillMessagesApi.sendTemplate(MandrillMessagesApi.java:206)
                         E      at com.microtripit.mandrillapp.lutung.controller.MandrillMessagesApi.sendTemplate(MandrillMessagesApi.java:133)
                         E      at org.getlantern.lantern.model.MailSender.sendMail(MailSender.java:32)
                         E      at org.getlantern.lantern.model.UI$11.doInBackground(UI.java:329)
                         E      at org.getlantern.lantern.model.UI$11.doInBackground(UI.java:322)
                         E      at android.os.AsyncTask$2.call(AsyncTask.java:288)
                         E      at java.util.concurrent.FutureTask.run(FutureTask.java:237)
                         E      at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1112)
                         E      at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:587)
                         E      at java.lang.Thread.run(Thread.java:841)
                         E  Caused by: com.microtripit.mandrillapp.lutung.model.HandleResponseException: Error handling Mandrill response
                         E      at com.microtripit.mandrillapp.lutung.model.MandrillRequest.handleResponse(MandrillRequest.java:74)
                         E      at com.microtripit.mandrillapp.lutung.model.MandrillRequestDispatcher.execute(MandrillRequestDispatcher.java:83)
                         E      ... 11 more
                         E  Caused by: java.lang.NoClassDefFoundError: org.apache.commons.io.IOUtils
                         E      at com.microtripit.mandrillapp.lutung.model.MandrillRequest.handleResponse(MandrillRequest.java:66)
                         E      ... 12 more
                         D  Showing alert dialog...
```

Comment (oxtoacart) [inline src/github.com/getlantern/lantern-mobile/app/build.gradle]: Cool.  Mind adding a comment to that effect?

Comment (atavism) [inline src/github.com/getlantern/lantern-mobile/app/build.gradle]: Sure. Just added it

Comment (oxtoacart) [inline src/github.com/getlantern/lantern-mobile/app/build.gradle]: Thanks!

**Comment by oxtoacart** [comment]
LGTM!
